projects/valtimo/object-management/src/lib/components/object-management-list/object-management-list.component.html

```diff
@@ -18,6 +18,10 @@
   <div class="container-fluid">
     <div class="col-12 px-0 mb-5">
       <div class="text-right mt-m3px mb-3">
+        <button (click)="showUploadModal()" class="btn btn-secondary btn-space">
```

Comment:
I found a problem. If I upload a objecttype without "id" field, it gives an error. But I'm not sure if this is a FE or BE issue.

Reply:
It should check for these keys, ['id', 'title', 'objecttypenApiPluginConfigurationId', 'objecttypeId', 'objectenApiPluginConfigurationId', 'showInDataMenu'], but maybe you left the "id" empty?

Reply:
I left the "id" field out. Llike this:
```
{
  "title": "Tree configuration",
  "objecttypenApiPluginConfigurationId": "e33997b5-08f7-11ed-a16c-9edfe5d3ca4c",
  "objecttypeId": "e33997b5-08f7-11ed-a16c-9edfe5d3ca4c",
  "objectenApiPluginConfigurationId": "e33997b5-08f7-11ed-a16c-9edfe5d3ca4c",
  "showInDataMenu": true,
  "formDefinitionView": null,
  "formDefinitionEdit": null
}
```

Because the "id" field is optional. Only these are required:
['title', 'objecttypenApiPluginConfigurationId', 'objecttypeId', 'objectenApiPluginConfigurationId', 'showInDataMenu']

Reply:
I see I thought it was required, I removed it now